templates/api.yaml

```diff
@@ -79,17 +88,893 @@ Resources:
             HttpMethod: '*'
         - !Ref AWS::NoValue
       DefinitionBody:
-        'Fn::Transform':
-          Name: 'AWS::Include'
-          Parameters:
-            Location: ./api.definition.yml
+        openapi: "3.0.1"
```

Comment:
oof, why the switch to have the API definition in the template? it makes this file massive. I assume it's something to do with conditions

Reply:
Yep, I tried many different ways to do this. But the previous Includes tranform we were using does not support CFn conditions. It only supports !Ref and !GetAtt so my hand was forced. If you think of anything that would get around this please shout i am as "happy" as you in having to do this.